pkl-doc/src/main/kotlin/org/pkl/doc/Main.kt

```diff
@@ -78,7 +81,7 @@ class DocCommand : BaseCommand(name = "pkldoc", helpLink = helpLink) {
       CliDocGeneratorOptions(
         baseOptions.baseOptions(modules, projectOptions),
         outputDir,
-        true,
+        isTestMode,
```

Comment:
Setting test mode to true disables some optimizations. For example, normally, we don't type-check properties or method return types on standard library (external) methods. However, we do type-check them when run with test mode on.